### ISSUE TITLE
Galaxy Effect Bug

### ISSUE THREAD

**Issue by Clyde-L** [smeighan/xLights#2375]
When adjusting the angle on the Galaxy effect, there is a "tail" added to the end of the effect. This is in 2020.45, but has been doing it for a while.  Just now getting around to posting. 

Thanks,

Clyde

![image](https://user-images.githubusercontent.com/12719816/100549616-00f1a980-3242-11eb-9663-0354d004cd99.png)

**Comment by keithsw1111** [comment]
No sample?

**Comment by AzGilrock** [comment]
Galaxy usually is one big tail


Sent from Nine
________________________________

**Comment by AzGilrock** [comment]
Ah...I couldn't see the photo when I was out hunting.  That looks more like a head remnant so I think I know what's going on.  It was a real pain to get that curved head drawn right so I bet the code that does all the rounding isn't taking that angle into account.

**Comment by AzGilrock** [comment]
So I found a couple issues.  The color for the head/tail wasn't right when the start angle was changed as well as the position issue.  Got both of those fixed.  The one final thing I'd like to change but didn't find a good method is when the width is larger than the current radius the code that draws the head/tail gets into an endless loop so there is protection to prevent that so what you will see is the head/tail doesn't appear when its close to center but then pops up when it gets further from the origin.  I feel like there should be a way to fix it but after 3 hours on this effect I just checked in the 2 fixes I had.  Thanks for finding this.
